grpcwebclient/impl.ts

```diff
@@ -0,0 +1,296 @@
+// Copyright 2023 The Connect Authors
```

Comment:
This is the actual client implementation (the equivalent of `impl.go` for the other Go clients). All the other ancillary TypeScript files are to communicate with the runner.